pkg/preflight/utils.go

```diff
@@ -29,10 +29,19 @@ func IsAWS() bool {
 	return Instance().ProviderName() == string(cloudops.AWS)
 }
 
+func IsVsphere() bool {
+	return Instance().ProviderName() == string(cloudops.Vsphere)
+}
+
+func IsPure() bool {
```

Comment:
Is this duplicate code? Since we are already setting cloudops.Pure, are we using this somewhere?

Reply:
These util functions were also for detecting provider when defaulting to dmthin, Will remove them as well.